de-lib/src/main/java/org/iplantc/de/apps/client/views/details/ReactCategoryTree.java

```diff
@@ -0,0 +1,28 @@
+package org.iplantc.de.apps.client.views.details;
+
+import org.iplantc.de.apps.client.AppDetailsView;
+
+import com.google.web.bindery.autobean.shared.Splittable;
+
+import gwt.react.client.components.ReactClass;
+import gwt.react.client.proptypes.BaseProps;
+import jsinterop.annotations.JsPackage;
+import jsinterop.annotations.JsProperty;
+import jsinterop.annotations.JsType;
+
+/**
+ * @author aramsey
+ */
+@JsType(isNative = true, namespace = "CyVerseReactComponents.apps.details", name = "CategoryTree")
+public class ReactCategoryTree {
+
+    @JsProperty(namespace = "CyVerseReactComponents.apps.details", name = "CategoryTree")
```

Comment:
I think the `@JsType` annotation on this class might be unnecessary.
It seems like you'd only need this `@JsProperty` annotation on the `CategoryTree` field.